tsconfig.json

```
@@ -21,7 +21,7 @@
     "resolveJsonModule": true,
     "outDir": "dist",
     "sourceMap": true,
-    "declaration": true,
+    "declaration": false,
```

This should be set to true.
```suggestion
    "declaration": true,
```